sensor/common/compliance/nodescan_handler_impl.go

```diff
@@ -0,0 +1,82 @@
+package compliance
+
+import (
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/generated/internalapi/central"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/centralsensor"
+	"github.com/stackrox/rox/pkg/concurrency"
+)
+
+type nodeScanHandlerImpl struct {
+	nodeScans <-chan *storage.NodeScanV2
+	toCentral chan *central.MsgFromSensor
+
+	stopC    concurrency.ErrorSignal
+	stoppedC concurrency.ErrorSignal
+}
+
+func (c *nodeScanHandlerImpl) Capabilities() []centralsensor.SensorCapability {
+	return []centralsensor.SensorCapability{centralsensor.NodeScanningCap}
+}
+
+func (c *nodeScanHandlerImpl) ResponsesC() <-chan *central.MsgFromSensor {
+	return c.toCentral
+}
+
+func (c *nodeScanHandlerImpl) Start() error {
```

Comment:
Do we want to allow multiple `Start` calls to happen?

Reply:
I investigated and this will cause a panic due to closing a channel twice (only when the feature-flag is enabled). I will provide a fix and a test for this.